x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_details/custom_query_rule.cy.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  getRulesManagementTableRows,
+  goToRuleDetailsOf,
+} from '../../../tasks/alerts_detection_rules';
+import { createRule } from '../../../tasks/api_calls/rules';
+import { SECURITY_DETECTIONS_RULES_URL } from '../../../urls/navigation';
+import { getDetails } from '../../../tasks/rule_details';
+import { getNewRule } from '../../../objects/rule';
+
+import {
+  CUSTOM_QUERY_DETAILS,
+  DEFINITION_DETAILS,
+  INDEX_PATTERNS_DETAILS,
+  RULE_NAME_HEADER,
+  RULE_TYPE_DETAILS,
+} from '../../../screens/rule_details';
+
+import { cleanKibana, deleteAlertsAndRules } from '../../../tasks/common';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+
+// Only testing components in the rule details page unique to rules using index patterns
+describe('Custom query rule', { tags: ['@ess', '@serverless'] }, () => {
+  const rule = getNewRule();
+
+  before(() => {
+    cleanKibana();
+  });
+
+  beforeEach(() => {
+    deleteAlertsAndRules();
+    createRule(rule);
+    login();
+    visitWithoutDateRange(SECURITY_DETECTIONS_RULES_URL);
+    getRulesManagementTableRows().should('have.length', 1);
+  });
+
+  it('Displays rule details', function () {
+    goToRuleDetailsOf(rule.name);
```

Comment:
Let's not test navigation from the Rule Management to the Rule Details page in this test, I think navigation should be tested separately. We can open the Rule Details page right away.